docs/.vuepress/config.js

```diff
@@ -12,23 +12,55 @@ const docsSidebar = [
     title: 'Documentation',
     collapsable: false,
     children: [
-      '/getting-started',
       {
-        title: 'Basic Features',
+        title: 'Introduction',
+        path: '/introduction/',
         collapsable: true,
         children: [
-          ['/basic-features/key_management', 'Key management'],
-          ['/basic-features/blockchain_data', 'Blockchain data'],
-          ['/basic-features/connecting_peers', 'Connecting peers'],
-          ['/basic-features/managing_channels', 'Managing channels'],
-          ['/basic-features/sending_payments', 'Sending payments'],
-          ['/basic-features/receiving_payments', 'Receiving Payments']
+          ['/introduction/use_cases', 'Use Cases'],
         ]
       },
+      '/running-a-sample-ldk-node',
+      {
+        title: 'Getting Started',
```

Comment:
Could this be moved after "Overview" in the navigation"? Some of the concepts used here might be confusing without first seeing the overall architecture and networking pages.

We may want to give this a better name, too. Not sure exactly what. Previously, it was called "Using LDK" but thinking about it a bit more, it seems to be more of a guide about payments -- connecting to peers and opening channels as a prerequisite and how to send and receive them.

Reply:
Yeah agreed.

How about `Making payments` or we also use the term `Common operations`?

Reply:
I suppose "Common Operations" (or similar) may be more appropriate given the first two items are needed to _route_ payments rather than only to _make_ payments. Or in that case maybe a bit more generic name around payments would be better? Simply "Payments"?